src/server/getters/get-fee-windows.ts

```diff
@@ -9,12 +9,15 @@ export function getFeeWindows(db: Knex, augur: Augur, universe: Address, account
   let query = db.select(["fee_windows.feeWindow", "fee_windows.startTime", "fee_windows.endTime", "balances.balance", "participation_token.supply AS participationTokenStake", "fee_token.supply AS feeTokenStake", "cash.balance as totalFees"]).from("fee_windows")
     .join("balances", "fee_windows.feeWindow", "balances.token")
     .join("token_supply AS participation_token", "participation_token.token", "fee_windows.feeWindow")
-    .join("token_supply AS fee_token", "fee_token.token", "fee_windows.feeToken")
-    .join("balances AS cash", "cash.token", augur.contracts.addresses[augur.rpc.getNetworkID()].Cash)
+    .leftJoin("token_supply AS fee_token", "fee_token.token", "fee_windows.feeToken")
+    .leftJoin("balances AS cash", function () {
+      this
+        .on("cash.token", augur.contracts.addresses[augur.rpc.getNetworkID()].Cash)
+        .on("cash.owner", db.raw("fee_windows.feeWindow"));
```

Comment:
is `db.raw` required here?

Reply:
It looks like it isn't strictly necessary, but doesn't hurt
```
    interface Join {
        (raw: Raw): QueryBuilder;
        (tableName: TableName, clause: (this: JoinClause) => void): QueryBuilder;
        (tableName: TableName, columns: { [key: string]: string | number | Raw }): QueryBuilder;
        (tableName: TableName, raw: Raw): QueryBuilder;

        (tableName: TableName, column1: string, column2: string): QueryBuilder;

        (tableName: TableName, column1: string, raw: Raw): QueryBuilder;
        (tableName: TableName, column1: string, operator: string, column2: string): QueryBuilder;
    }

```

Reply:
Interesting. FWIW tests will fail without it but succeed if I substituted the string of the actual address.